bcda/auth/ssas_test.go

```diff
@@ -24,7 +26,10 @@ func (s *SSASPluginTestSuite) TestDeleteSystem() {}
 
 func (s *SSASPluginTestSuite) TestResetSecret() {}
 
-func (s *SSASPluginTestSuite) TestRevokeSystemCredentials() {}
+func (s *SSASPluginTestSuite) TestRevokeSystemCredentials() {
+	err := s.p.RevokeSystemCredentials("a-fake-ssasID")
```

Comment:
I'm not sure we're testing something useful here.  Without wiring up or mocking SSAS, we will indeed always have an error, but not one that exercises the (admittedly minimal) code.